selfdrive/car/honda/fingerprints.py

```diff
@@ -395,6 +395,46 @@
       b'77959-TMM-F040\x00\x00',
     ],
   },
+  CAR.HONDA_ODYSSEY_BOSCH: {
+    (Ecu.programmedFuelInjection, 0x18da10f1, None): [
+      b'37805-5MR-C730\x00\x00',  # 2021 Touring
+      b'37805-5MR-C910\x00\x00',  # 2022 Touring
+      b'37805-5MR-C920\x00\x00',  # 2022 Elite
+      b'37805-5MR-C110\x00\x00',  # 2023 Sport
```

Comment:
Remove the trailing comments on this and all other firmware, they're not really possible to maintain.

```suggestion
      b'37805-5MR-C730\x00\x00',
      b'37805-5MR-C910\x00\x00',
      b'37805-5MR-C920\x00\x00',
      b'37805-5MR-C110\x00\x00',
```